Tests/Pcap++Test/Tests/FileTests.cpp

```diff
@@ -26,6 +26,204 @@ class FileReaderTeardown
 	}
 };
 
+PTF_TEST_CASE(TestFileFormatDetector)
```

Comment:
Please see my previous comment. Maybe we can create a temp fake file with the expected data and run `createReader()`